dev/Command/RegistrationCommand.php

```diff
@@ -110,22 +111,21 @@ protected function execute(InputInterface $input, OutputInterface $output)
         if ($resultBody !== 'OK' || $result->getStatusCode() !== 200) {
             $output->writeln('<error>Enrollment failed</error>');
 
-            return;
+            return 1;
         }
         $output->writeln('<info>Enrollment succeeded</info>');
 
         // Storing result as a new identity.
         $this->storeIdentity($metadata, $secret, $output);
+        return 0;
     }
 
-    protected function decorateResult($text)
+    protected function decorateResult($text): string
     {
         return "<options=bold>$text</>";
     }
 
     /**
-     * @param OutputInterface $output
```

Comment:
Was this supposed to get removed if it is still a parameter of readRegistrationUrlFromFile ?

Reply:
I don't see that @param in my codebase. Could you have been commenting on an older version of this file? I did however made the rest of this class more type safe. So there's that.